test/integration/test___main__.py

```diff
@@ -209,7 +209,10 @@ def test_cmdline_playbook_hosts():
         with open(inventory, 'w') as f:
             f.write('[all]\nlocalhost ansible_connection=local')
 
-        cmdline('run', private_data_dir, '-p', playbook, '--hosts', 'all')
+        # privateip: removed --hosts command line option from test beause it is
```

Comment:
Is it covered in some other tests? We may want to add a test specifically for this.

Reply:
I need to add a couple of new test cases... there are one or two more scenarios i realized i didn't account for yet ... updated PR coming soon.